pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -190,6 +185,23 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig, enableProxy, enable
 		return nil, err
 	}
 
+	// Initialize SDN before building kubelet config so it can modify options
```

Comment:
We want all of the server initialization code together - that's a commonly reviewed bit of code, and if someone comes in here now they will miss it.